### PR TITLE
GRAM-105: Fix for S hemisphere Polar coordinate

### PR DESCRIPTION
**Pull request by tomwang-work**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 *.class
+classes/images
```
Comment (tomwang-work) [inline .gitignore:3]: Just to help cut down on random files showing up potentially for adding repository

File: src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java [hunk 2]
```diff
@@ -1103,7 +1103,7 @@ private UPSCoord convertMGRSToUPS(String MGRS)
             {
                 hemisphere = AVKey.SOUTH;
 
-                ltr2_low_value = upsConstants[mgrs.latitudeBand][12]; //.ltr2_low_value;
+                ltr2_low_value = upsConstants[mgrs.latitudeBand][1]; //.ltr2_low_value;
```
Comment (tomwang-work) [inline src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java:1106]: Clearly a typo. See the pattern above, too. This is causing `IndexOutOfBoundsException`. upsConstants is a 2-dimensional array whose second dimension all have 6 elements.

File: test/gov/nasa/worldwind/geom/coords/MGRSCoordTest.java [hunk 3]
```diff
@@ -24,5 +24,15 @@ public void testPolar() {
            // expectedMGRS.toString().
            assertEquals(expectedMGRS.toString().replaceAll(" ", ""), actualMGRS.toString());
        }
+        public void testSPolarFromString() {
```
Comment (tomwang-work) [inline test/gov/nasa/worldwind/geom/coords/MGRSCoordTest.java:27]: Confirm my hypothesis on typo.

File: src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java (hunk 2 above)
Comment (qnix) [reply src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java:1106]: Wow, really!  Nice catch!

**Review by qnix — APPROVED** [review]
Looking good to me. 👍

**Comment by tomwang-work** [comment]
Thanks :)

**Pull request merged**